src/bundle/Resources/config/services/tabs/locationview.yml

```diff
@@ -1,11 +1,24 @@
 services:
     EzSystems\EzPlatformAdminUi\Tab\LocationView\:
         resource: "../../../lib/Tab/LocationView/*"
+        exclude: '../../../lib/Tab/LocationView/PagerLocationToDataMapper.php'
         parent: EzSystems\EzPlatformAdminUi\Tab\AbstractTab
         public: true
         tags:
             - { name: ezplatform.tab, group: location-view }
 
+    EzSystems\EzPlatformAdminUi\Tab\LocationView\PagerLocationToDataMapper:
+        autowire: true
+        public: false
+
+    EzSystems\EzPlatformAdminUi\Tab\LocationView\LocationsTab:
+        parent: EzSystems\EzPlatformAdminUi\Tab\AbstractTab
+        public: true
+        arguments:
+            $defaultPaginationLimit: '$pagination.location_limit$'
```

Comment:
This is a dynamic setting. It's deprecated and is not going to work in upper branches. Use ConfigResolver on demand instead.